metadata/tests/test_examples.py

```diff
@@ -246,3 +248,249 @@ def test_sticky_status_is_set(self):
         self.assertTrue(submission_data['finalized'], "Should be finalized")
         self.assertEqual(submission_data['status'], 'hold',
                          "Status should be `hold`, as sticky_status was set")
+
+
+class TestPublicationIntegration(TestCase):
```

Comment:
These are the same tests as in core/events, but expressed at the level of the API